libcaf_net/caf/policy/scribe.hpp

```diff
@@ -0,0 +1,160 @@
+/******************************************************************************
+ *                       ____    _    _____                                   *
+ *                      / ___|  / \  |  ___|    C++                           *
+ *                     | |     / _ \ | |_       Actor                         *
+ *                     | |___ / ___ \|  _|      Framework                     *
+ *                      \____/_/   \_|_|                                      *
+ *                                                                            *
+ * Copyright 2011-2018 Dominik Charousset                                     *
+ *                                                                            *
+ * Distributed under the terms and conditions of the BSD 3-Clause License or  *
+ * (at your option) under the terms and conditions of the Boost Software      *
+ * License 1.0. See accompanying files LICENSE and LICENSE_ALTERNATIVE.       *
+ *                                                                            *
+ * If you did not receive a copy of the license files, see                    *
+ * http://opensource.org/licenses/BSD-3-Clause and                            *
+ * http://www.boost.org/LICENSE_1_0.txt.                                      *
+ ******************************************************************************/
+
+#pragma once
+
+#include <caf/error.hpp>
+#include <caf/fwd.hpp>
+#include <caf/logger.hpp>
+#include <caf/net/endpoint_manager.hpp>
+#include <caf/net/receive_policy.hpp>
+#include <caf/net/stream_socket.hpp>
+#include <caf/sec.hpp>
+#include <caf/span.hpp>
+#include <caf/variant.hpp>
+
+#ifdef CAF_WINDOWS
+#  include <winsock2.h>
```

Comment:
Should *never* get included in a `.hpp` file. Also, this entire include block seems unnecessary.